### PR TITLE
[#33] Added a README

### PR DESCRIPTION
**Pull request by yourbuddyconner**
Additions: 
- Added a README with loose instructions for setting up a testnet, should be updated regularly as code changes.

### CODE REVIEW

**Comment by jkrauska** [comment]
@yourbuddyconner cool -- can you scan https://github.com/o1-labs/testnet and see if any of that material makes sense to add here?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,153 @@
-# coda-automation
-Tools for managing coda testnets
+![Coda Logo](https://cdn.codaprotocol.com/v4/static/img/coda-logo.png)
+
+# Repository Purpose 
+This repository is designed to show an opinionated example on how to operate a network of Coda Daemons. It implements the entire node lifecycle using a modern Infrastructure as Code toolset. Community contributions are warmly encouraged, please see the [contribution guidelines](#to-do) for more details. The code is designed to be as modular as possible, allowing the end-user to "pick and choose" the parts they would like to incorporate into their own infrastructure stack.
```
Comment (o1pranay) [inline README.md]: would prefer to use word "nodes" over "daemons" - what do you think?

also, good call to add a link for contribution guidelines - could just throw in a barebones version of https://github.com/CodaProtocol/coda/blob/master/CONTRIBUTING.md for now while we don't have anything

File: README.md [hunk 2]
```diff
@@ -1,2 +1,153 @@
...
+    │   └── elasticsearch
+    └── testnets
+        ├── testnet20190601
+        └── testnet20190613
+```
+
+**Terraform:** Contains resource modules and live code to deploy a Coda Testnet. 
+- Note: Currently **ALL** modules only support AWS, multi-cloud support is on the roadmap.
+- *coda-node:* A Terraform module that encapsulates a single node.
+- *elasticsearch:* A Terraform module that deploys an elasticsearch cluster using the AWS Elasticsearch service.
```
Comment (o1pranay) [inline README.md]: would it help to add a link to AWS ES? https://aws.amazon.com/elasticsearch-service/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,153 @@
...
+  source        = "../../modules/coda-node"
+  region        = "us-west-2"
+  server_count  = 1
+  instance_type = "c5.xlarge"
+  netname       = "${local.netname}"
+  rolename      = "seed"
+  public_key    = "${local.public_key}"
+}
+
+## One Snarker
```
Comment (o1pranay) [inline README.md]: maybe we can explain what these things are in a glossary before this - or refer to a global glossary once we make one!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,153 @@
...
+  netname       = "${local.netname}"
+  rolename      = "proposer"
+  public_key    = "${local.public_key}"
+}
+```
+
+If you were to `terraform apply` this, you would get 5 instances, all in AWS's `us-west-2` region. However there are no Coda daemons installed! 
+
+### Run Ansible against your Coda Testnet
+Next, we need to install the coda daemon.
```
Comment (o1pranay) [inline README.md]: nit: capitalize Coda

File: README.md [hunk 5]
```diff
@@ -1,2 +1,153 @@
...
+$ ansible-playbook -i ec2.py -u admin -e netname=$(date +"%Y%m%d") coda-start.yaml
+```
+
+This step will start the seed node, then join any Snarkers to the network before joining any Proposers. 
+
+## Join Your Coda Nodes to an Existing Test Network
+
+**TO DO**
+
+# Next Steps
```
Comment (o1pranay) [inline README.md]: let's move the callback to go to discord into `Repository Purpose`
maybe a final sentence -

If you have any issues setting up your testnet with the instructions below, join the public Discord Server and get help from the Coda community.

^^^ feel free to reword this however you want!

File: README.md (hunk 2 above)
Comment (yourbuddyconner) [reply README.md]: Yes 100%

File: README.md (hunk 3 above)
Comment (yourbuddyconner) [reply README.md]: Yeah, I have a "Key Terms" section where I am defining vocabulary specific to this repo, but the documentation would benefit from a central glossary in the Coda repository as a source of truth.

File: README.md (hunk 1 above)
Comment (yourbuddyconner) [reply README.md]: Yeah, I think here I'd like to defer to a central glossary of terms as well as a global contribution guidelines page. Doesn't make sense for each project repo to have its own set of guidelines.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,153 @@
...
+Now that you have a testnet running, there's plenty of things you can do: 
+- Let us know! Join the public [Discord Server](https://discord.gg/ShKhA7J). 
+- Fund your wallets -- Sign up for the [testnet faucet](#to-do)!
```
Comment (Schmavery) [inline README.md]: Cool, but maybe some of this will be confusing before they actually lead somewhere

**Pull request merged**